source/libs/page-detect.ts

```diff
@@ -363,6 +363,11 @@ export const _isTrending = [
 	'https://github.com/trending/unknown'
 ];
 
+export const isBranches = (): boolean => /^branches/.test(getRepoPath()!);
+export const _isBranches = [
+	'https://github.com/sindresorhus/refined-github/branches'
```

Comment:
You can also add:
- `'https://github.com/sindresorhus/branches'` to `isRepoRoot`